File: mifosng-android/src/main/java/com/mifos/api/model/SaveResponse.java
/*
 * This project is licensed under the open source MPL V2.
 * See https://github.com/openMF/android-client/blob/master/LICENSE.md
 */

package com.mifos.api.model;


import com.google.gson.Gson;

public class SaveResponse {
    public int groupId;
    public int resourceId;
    public Changes changes;

    @Override
    public String toString() {
        return new Gson().toJson(this);
    }

}

File: core/database/src/main/java/com/mifos/core/objects/response/SaveResponse.kt
/*
 * Copyright 2024 Mifos Initiative
 *
 * This Source Code Form is subject to the terms of the Mozilla Public
 * License, v. 2.0. If a copy of the MPL was not distributed with this
 * file, You can obtain one at https://mozilla.org/MPL/2.0/.
 *
 * See https://github.com/openMF/android-client/blob/master/LICENSE.md
 */
package com.mifos.core.objects.response

import com.google.gson.Gson
import com.google.gson.annotations.SerializedName
import com.mifos.core.objects.Changes

class SaveResponse {
    @SerializedName("groupId")
    var groupId: Int? = null

    @SerializedName("resourceId")
    var resourceId: Int? = null

    @SerializedName("officeId")
    var officeId: Int? = null
    var changes: Changes? = null
    override fun toString(): String {
        return Gson().toJson(this)
    }
}
